pages/linux/runsv.md

```diff
@@ -0,0 +1,11 @@
+# runsv
+
+> Start and manage a runit service.
+
+- Start a runit service as yourself:
```

Comment:
"as the current user" might sound better.

Reply:
Fixed